setup.py

```diff
@@ -35,5 +35,6 @@ def get_version(rel_path):
     install_requires=[
         "jax==0.2.7",
         "jaxlib==0.1.57",
+        "numpy==1.19.4",
```

Comment:
just to clarify - will these only be pinned so tightly during development?

Reply:
Numpy can probably be relaxed. JAX version will need to be bumped manually however. The library is immature and can introduce breaking changes at any moment. Happened to me several times, I am sure @fehiepsi and @neerajprad can relate.

Reply:
Yes, unfortunately, this is true. We didn't pin the jax and jaxlib versions to begin with but that would force us to update every time jax released a new version because that would break things internally. Until jax is more mature and has a predictable release cycle, it will be best to pin.